NumLib/TimeStepping/Algorithms/EvolutionaryPIDcontroller.h

```diff
@@ -91,6 +91,10 @@ class EvolutionaryPIDcontroller final : public TimeStepAlgorithm
     /// Get a flag to indicate that this algorithm need to compute
     /// solution error.
     bool isSolutionErrorComputationNeeded() override { return true; }
+    /// Add specific times
+    void addSpecificTimes(
```

Comment:
It would be better to add comment on what this function does aside the info given in the function's name.

Reply:
Added a more detailed description.